package.json

```diff
@@ -30,12 +30,15 @@
     "babel-loader": "^8.2.2",
     "babel-plugin-syntax-dynamic-import": "^6.18.0",
     "babel-preset-es2015": "^6.24.1",
+    "copy-webpack-plugin": "^9.0.1",
     "css-loader": "^5.1.2",
     "css-minimizer-webpack-plugin": "^1.3.0",
     "jest": "^26.6.3",
     "mini-css-extract-plugin": "^1.3.9",
+    "source-map-loader": "^3.0.0",
     "style-loader": "^2.0.0",
     "terser-webpack-plugin": "^5.1.1",
+    "ts-loader": "^9.2.5",
     "webpack": "^5.25.1",
     "webpack-cli": "^4.5.0",
     "webpack-dev-server": "^3.11.2"
```

Comment:
Looks like you have Typescript installed globally on your system? May be worth adding the version of typescript you're on as a dev dependency.